notebooks/04-non-sequential.ipynb

```diff
@@ -2,73 +2,63 @@
  "cells": [
```

Comment:
<p>Can you make the next bullet point smaller?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
It is only big in reviewnb..

Reply:
does that mean, I should not make it smaller?

You have write access, feel free to change it.

Reply:
Leave it as it is. It loooks good :)